core/rend/gl4/gles.cpp

```diff
@@ -662,14 +662,15 @@ static bool RenderFrame()
 		Handle Dc to screen scaling
 	*/
 	float screen_scaling = is_rtt ? 1.f : settings.rend.ScreenScaling / 100.f;
+	float screen_stretching = settings.rend.ScreenStretching / 100.f;
```

Comment:
I'd add this as a more generic screen scaling argument - we can do various aspect ratios, considering the NTSC safe area and such, not only anamorphic widescreen

Reply:
yes, we could add a vertical stretching option as well, positive or negative

Reply:
See https://github.com/emudev-org/nulldce/blob/39c05500629116f0eb6bcb89a94027e1b987170c/plugs/drkPvr/softRend.cpp#L345 for good values